test.adoc

```diff
@@ -0,0 +1,7 @@
+Don't do this.
+
+You can certainly do this.
+
+However, never forget how great OpenShift Container Platform is.
+
+Adding another line for commit 2 ABC.
```

Comment:
Define acronyms and abbreviations (such as 'ABC') on first occurrence if they're likely to be unfamiliar.